src/Stratis.Bitcoin.Features.Wallet.Tests/WalletControllerTest.cs

```diff
@@ -2351,16 +2320,16 @@ public void RemoveTransactionsWithIdsRemovesAllTransactionsByIds()
             // Arrange.
             string walletName = "wallet1";
             Wallet wallet = WalletTestsHelpers.CreateWallet(walletName);
-            wallet.AccountsRoot.Add(new AccountRoot());
+
             uint256 trxId1 = uint256.Parse("d6043add63ec364fcb591cf209285d8e60f1cc06186d4dcbce496cdbb4303400");
             var resultModel = new HashSet<(uint256 trxId, DateTimeOffset creationTime)>();
             resultModel.Add((trxId1, DateTimeOffset.Now));
 
             var walletManager = new Mock<IWalletManager>();
             var walletSyncManager = new Mock<IWalletSyncManager>();
             walletManager.Setup(manager => manager.RemoveTransactionsByIds(walletName, new[] { trxId1 })).Returns(resultModel);
-            walletManager.Setup(manager => manager.GetWallet(walletName)).Returns(wallet);
-            walletSyncManager.Setup(manager => manager.SyncFromHeight(It.IsAny<int>()));
+            //walletManager.Setup(manager => manager.GetWalletByName(walletName)).Returns(wallet);
```

Comment:
Please remove or add comment on why this commented out.

Reply:
Removed. Not required.